WordPress/Classes/ViewRelated/Reader/Detail/Views/ReaderDetailCommentsTableViewDelegate.swift

```diff
@@ -114,13 +114,31 @@ class ReaderDetailCommentsTableViewDelegate: NSObject, UITableViewDataSource, UI
         return header
     }
 
+    func tableView(_ tableView: UITableView, viewForFooterInSection section: Int) -> UIView? {
+        guard section == 0, JetpackBrandingVisibility.all.enabled else {
```

Comment:
can we add a comment on why/how we are using the section footer instead of the table view footer? Just in case in the future we need to add sections

Reply:
Good call - added in 2653864e929c06397e320830f700eb2092777f63.